classes/Http/Form/ForgotForm.php

```diff
@@ -17,13 +17,13 @@
 use Symfony\Component\Form\Extension\Core\Type\EmailType;
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
 use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator;
```

Comment:
It's actually a common practice to import Validator constraints with alias `Assert`. 😕